packages/core/src/internal/utils/environment.ts

```diff
@@ -0,0 +1,9 @@
+import isNil from 'ramda/es/isNil.js';
```

Comment:
We have our own `isNil` that can be imported from `identity.js`...

We added it because the ramda imports were going circular on us...